clarity/src/vm/costs/mod.rs

```diff
@@ -865,6 +865,9 @@ fn compute_cost(
 
 fn add_cost(s: &mut TrackerData, cost: ExecutionCost) -> std::result::Result<(), CostErrors> {
     s.total.add(&cost)?;
+    if cfg!(feature = "disable-costs") {
```

Comment:
This is kind of a hack... maybe add a todo or an issue to remove it, longer term?

Reply:
Agreed, a comment is needed!